doc/examples/rp.rst

```diff
@@ -48,10 +48,10 @@ you will learn what information to look for.
 Issuer discovery
 ----------------
 
-OIDC uses `webfinger`_ to to do the OP discovery. In very general terms this means
+OIDC uses `webfinger`_ to do the OP discovery. In very general terms this means
 that the user that accesses the RP provides an identifier. There are a number
 of different syntaxes that this identifier can adhere to. The most common is
-probably the e-mail address syntax. It's something that looks like an e-mail
+probably the email address syntax. It's looks like an email
```

Comment:
"It looks like an email" not "It's"

Reply:
Ah, good catch. I will quickly change it.

Reply:
I pushed a change.